### ISSUE TITLE
Database migrations fail

### ISSUE THREAD

**Issue by SamR1** [SamR1/FitTrackee#504]
The following error appears when migrations are applied with a **fresh** install with the package (no problem with installation from source):

```shell
Traceback (most recent call last):
  File "/usr/local/bin/ftcli", line 8, in <module>
    sys.exit(cli())
             ^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/fittrackee/migrations/commands.py", line 24, in upgrade_db
    upgrade(directory=BASEDIR)
  File "/usr/local/lib/python3.11/site-packages/flask_migrate/__init__.py", line 111, in wrapped
    f(*args, **kwargs)
  File "/usr/local/lib/python3.11/site-packages/flask_migrate/__init__.py", line 198, in upgrade
    config = current_app.extensions['migrate'].migrate.get_config(directory,
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/flask_migrate/__init__.py", line 96, in get_config
    for x in g.x_arg:
             ^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/flask/ctx.py", line 56, in __getattr__
    raise AttributeError(name) from None
AttributeError: x_arg
```

It seems to be linked with the new version of **Flask-Migrate** (4.0.6), released on March 09, 2024.

Until a fix is released, the workaround is to install the previous version:

```shell
$ pip install Flask-Migrate==4.0.5
```

**Comment by SamR1** [comment]
release in v0.7.32

**Comment by SamR1** [comment]
A new version of **Flask-Migrate** has been released (4.0.7), fixing the error on missing attribute.
